services/gundeck/src/Gundeck/Push.hs

```diff
@@ -413,8 +413,10 @@ addToken :: UserId -> ConnId -> PushToken -> Gundeck AddTokenResponse
 addToken uid cid newtok = mpaRunWithBudget 1 AddTokenNoBudget $ do
   (cur, old) <- foldl' (matching newtok) (Nothing, []) <$> Data.lookup uid Data.LocalQuorum
   Log.info $
-    "user" .= UUID.toASCIIBytes (toUUID uid)
-      ~~ "token" .= Text.take 16 (tokenText (newtok ^. token))
+    "user"
+      .= UUID.toASCIIBytes (toUUID uid)
+      ~~ "token"
+      .= Text.take 16 (tokenText (newtok ^. token))
```

Comment:
is this hlint telling us "no parens" and then ormolu going "oh, sweet, no parens, i can re-layout this!"?  oh dear.  i'm suddenly in favor of removing that hlint rule globally.

Reply:
I don't like it either